### DISCUSSION TITLE
Tailwindcss Parser

### DISCUSSION THREAD

**Discussion by graciasc** [remarkjs/remark#1041] [General] [answered]
Does remark parse tailwindcss utility classes ? if not this would be a really cool feature to at least add to existing application that use `remark-parse`

**Comment by remcohaszing** [comment]
I don’t see how remark and Tailwindcss are related. What do you mean by _parsing Tailwindcss utility classes_? And what do you envision it would look like in remark?

**Reply by graciasc** [reply]
Doesn't `remark parse` parse html and css in markdown ?

**Reply by ChristianMurphy** [reply]
`remark-parse` does not.
HTML inside of markdown is treated as its own embedded document and is ignored/left in a raw form by any commonmark compatible parser.

As @wooorm notes in https://github.com/orgs/remarkjs/discussions/1041#discussioncomment-3633011
Rehype would be needed to handle HTML embeds.

**Comment by wooorm** [comment, marked as answer]
Sounds a bit like this: https://github.com/jakobo/codedrift/issues/103 maybe?

Also: you should likely do it in rehype (which is where things happen on HTML), not in remark (which is where things happen on markdown)
